Sources/ScipioKit/ProjectGenerator.swift

```diff
@@ -62,19 +100,21 @@ struct ProjectGenerator {
         return .init(project: project, projectPath: projectPath)
     }
 
-    private func buildDistributionXCConfigContents(embedDebugSymbols: Bool) -> String {
-        let base = """
-        BUILD_LIBRARY_FOR_DISTRIBUTION = YES
-        ENABLE_BITCODE = YES
-        OTHER_CFLAGS = -fembed-bitcode
-        """
-        if embedDebugSymbols {
-            let debugSymbol = """
-        DEBUG_INFORMATION_FORMAT = dwarf-with-dsym
-        """
-            return [base, debugSymbol].joined(separator: "\n")
+    private func makeXCConfigData(shouldEmbedDebugSymbols: Bool, isStaticFramework: Bool) -> Data {
```

Comment:
why `should`?

Reply:
or `isEmbeddedDebugSymbols`

Reply:
i think `is` is better since there is no way to disable embed despite it is true